src/java/org/apache/cassandra/repair/RepairRunnable.java

```diff
@@ -141,11 +140,14 @@ protected void fireErrorAndComplete(int progressCount, int totalProgress, String
     static class CommonRange
     {
         public final Set<InetAddressAndPort> endpoints;
```

Comment:
that's not the case. transEndpoints is a subset of endpoints

Reply:
Then it needs to be allEndpoints or something. The relationship to transEndpoints is not obvious.

Reply:
And if you could do it all the way down starting at submitRepair session. I mean I know you might as well assume endpoints means allEndpoints, but it's definitely easier for me to understand that this code was touched by transient replication when it says allEndpoints.

Reply:
That's also not accurate, since the repair coordinator (local) endpoint is usually not included in that collection. I think just `endpoints` is probably informative enough, especially since we assert that transEndpoints is a subset of endpoints in at least a few places.